core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_prowconfig.yaml

```diff
@@ -36,3 +54,19 @@ tide:
     - needs-rebase
     repos:
     - openshift/coredns-ocp-dnsnameresolver
+  - includedBranches:
+    - release-4.17
```

Comment:
You didn't add the config of `release-4.17` branch but added the lables requirements, why?

Reply:
I actually am not sure whether the config for release-4.17 and 4.18 will automatically get added or they are needed to be added manually. I think I can probably remove the label requirements for 4.17 as I have not added the config for it.

Reply:
I have removed the label requirements for release-4.17.